server/channels/api4/plugin_test.go

```diff
@@ -2087,7 +2155,10 @@ func TestPluginWebSocketSession(t *testing.T) {
 	require.NoError(t, err)
 	backend := filepath.Join(pluginDir, pluginID, "backend.exe")
 	utils.CompileGo(t, string(pluginCode), backend)
-	os.WriteFile(filepath.Join(pluginDir, pluginID, "plugin.json"), []byte(`{"id": "`+pluginID+`", "server": {"executable": "backend.exe"}}`), 0600)
+	err = os.WriteFile(filepath.Join(pluginDir, pluginID, "plugin.json"), []byte(`{"id": "`+pluginID+`", "server": {"executable": "backend.exe"}}`), 0600)
+	if err != nil {
+		t.Fatalf("failed to write plugin.json file: %v", err)
+	}
```

Comment:
Please use `require.NoError(t, err)` instead

Reply:
Fixed this!